src/main/java/frc/robot/Constants.java

```diff
@@ -80,9 +81,21 @@ public static final class CounterweightConstants {
         public static final int ENDSTOP_SWITCH_PORT = 0;
         public static final int[] ENCODER_DIO_PINS = {1, 2}; // Encoder wired into DIO pins 1 & 2 on the roboRIO
     }
-    public static final class ArmConstants {
-        public static final int ROTATE_CAN_ID=0;
-        public static final int TELESCOPE_CAN_ID=0;
+    public static final class IntakeConstants {
+        public static final int ARM_LIFT_CAN_ID=42;
```

Comment:
i think rotate is better than lift becuase the arm rotates and lift is kinda ambigous